prin

```diff
@@ -53,7 +53,13 @@ PROG=${0##*/}
 # If NO ARGUMENTS should return *usage*, uncomment the following line:
 usage=${1:-yes}
 
-source $(dirname $(readlink -ne $BASH_SOURCE))/lib/common.sh
+if [[ $(uname) == "Darwin" ]]; then
```

Comment:
I find I can't run this with my >4.0 version of bash as installed by homebrew unless I invoke as `bash ./prin` or change the hashbang to `#!/usr/bin/env bash`

Reply:
There doesn't appear to be a check for whether `envsubst` exists

Reply:
Thanks @spiffxp, That is real good suggestion. I will update it

Reply:
`envsubst` isn't present by default on darwin, can you check if it's present and error out if not?

Reply:
@spiffxp  Yea. added the check action here. please review this. Thanks for your great efforts.